src/containers/layout/sidebar/Sidebar.tsx

```diff
@@ -4,35 +4,61 @@ import { useTranslation } from 'react-i18next'
 import Typography from '@mui/material/Typography'
 import List from '@mui/material/List'
 import ListItem from '@mui/material/ListItem'
+import Divider from '@mui/material/Divider'
+import Box from '@mui/material/Box'
 
 import HashLink from '~/components/hash-link/HashLink'
+import useBreakpoints from '~/hooks/use-breakpoints'
 
 import { styles } from '~/containers/layout/sidebar/Sidebar.styles'
 
+interface RouteItem {
```

Comment:
move it to common interfaces so we could reuse it

Reply:
fixed